src/js/components/Data/Data.js

```diff
@@ -128,6 +128,7 @@ export const Data = ({
       // longer reflecting the view
       delete nextView.name;
       delete nextView.view;
+      delete nextView.columns;
```

Comment:
```suggestion

```

Reply:
I don't beleive we need this..
Otherwise looks good

Reply:
I would agree with removing this. The "Clear filters" button should be scoped to clearing out filters. That said, I wouldn't expect that clearing my filters would also reset the visibility of columns.